packages/core/content-type-builder/admin/src/components/Relation/RelationNaturePicker/RelationNaturePicker.tsx

```diff
@@ -108,8 +108,11 @@ export const RelationNaturePicker = ({
                     }}
                     padding={2}
                     type="button"
+                    aria-label={formatMessage({ id: getTrad(`relation.${relation}`) })}
```

Comment:
This actually leads to a duplicate of "has many" where one is the inverted has many, but I didn't want to add a new translation for it. If someone has a good idea let me know -- like if we have another string somewhere that we can use that explains it better